content/v1.0.0/index.pl.md

```diff
@@ -0,0 +1,186 @@
+---
+draft: false
+aliases: ["/pl/"]
+---
+
+# Conventional Commits 1.0.0
+
+## Podsumowanie
+
+Specyfikacja Conventional Commits to lekka konwencja dotycząca wiadomości commitów.
+Zapewnia prosty zestaw reguł do tworzenia przejrzystej historii commitów;
+co ułatwia budowanie narzędzi automatyzujących na jej podstawie.
+Ta konwencja współgra z [SemVer](http://semver.org),
+opisując funkcje, poprawki i breaking changes w wiadomościach commitów.
+
+Wiadomość commita powinna być zbudowana w następujący sposób:
+
+---
+
+```
+<type>[opcjonalny scope (zakres)]: <opis>
+
+[opcjonalny body (treść)]
+
+[opcjonalny footer (stopka)]
+```
+
+---
+
+<br />
+Commit zawiera następujące elementy strukturalne, aby przekazać intencję
+otrzymującym Twoją bibliotekę:
```

Comment:
```suggestion
używającym Twojej biblioteki:
```

It's really hard to find a better word for `library` here :/ It does not fully match what I would like it to sound.